apps/SampleSite/js/themeeditor.js

```diff
@@ -564,4 +592,47 @@ function g_defaultSerials(idx,doReset) {
     } else {
         localStorage.themeEditorSerials = JSON.stringify(defaultSerials);
     }
+}
+function getProperName(name) {
```

Comment:
I agree but they have no common js files...  Should I make a new js file to include with just this?  Didn't seem like enough code, although moving forward it will be useful for new pages that have menus of all widgets...  Perhaps even creating a single menu-building script that can just be included in any new page